crates/sui-framework/packages/bridge/sources/limiter.move

```diff
@@ -65,6 +64,54 @@ module bridge::limiter {
         }
     }
 
+    struct UpdateRouteLimitEvent has copy, drop {
+        sending_chain: u8,
+        receiving_chain: u8,
+        new_limit: u64,
+    }
+
+    struct UpdateAssetPriceEvent has copy, drop {
+        token_id: u8,
+        new_price: u64,
+    }
+
+    // Abort if the route limit is not found
+    public fun get_route_limit(self: &TransferLimiter, route: &BridgeRoute): &u64 {
```

Comment:
Nit: on this and the below function any reason to return a `&u64`? Generally we don't return numeric types by reference.